airflow/provider.yaml.schema.json

```
@@ -167,6 +167,13 @@
           "python-module"
         ]
       }
+    },
+    "extra-links": {
+      "type": "array",
```

We can define a (basic) pattern at the schema level too:

```suggestion
      "type": "array",
       "uniqueItems": true,
       "pattern": "^\.[a-zA-Z]+[a-zA-Z0-9.\\-]*"
```

---

Yeah. I had it initially, there are a few more places where we can add uniqueness/regexp rules. So maybe we should do it as  a separate PR once we merge those. By that time we will have all the values in and tests to check if all our regexps/rules are valid